### ISSUE TITLE
Employer NI savings contributed to pension

### ISSUE THREAD

**Issue by wozniakpawel** [wozniakpawel/cooltaxtool#29]
Some employers allow for NI savings to be contributed back to pension. Add this as an option (switch)

**Comment by jrothlis** [comment]
Hi @wozniakpawel, just checking this is the same as what my employer does: if I do a bonus sacrifice, they will increase the amount by 14.3%, so if I sacrifice £20,000 of my bonus, I will actually receive £22,860 in my SIPP. If this isn't what you mean, could you add an option to allow that 14.3% uplift? Thanks for a fantastic tool!

**Comment by wozniakpawel** [comment]
Hi @jrothlis it probably is the same thing, although 14.3% sounds a little bit off. I would expect the NI savings to not go above 12%, but it will depend on the tax year in question. For example, for [2023/24](https://www.gov.uk/national-insurance-rates-letters) it could be up to 13.8%.
Do you have any more information on how that works for your employer? Maybe the 14.3% is actually the NI savings (let's say 13.8%) and some fixed amount independent of your bonus sacrifice amount?

**Comment by jrothlis** [comment]
According to this https://parasolgroup.co.uk/help-me-decide/guides/umbrella-payslips/ - 13.8% is for Employer’s National Insurance contributions and the remaining 0.5% relates to the apprentice levy, which all businesses in the UK with a payroll bill of more than £3,000,000 must pay.

**Comment by wozniakpawel** [comment]
I see! That's interesting, and makes a lot of sense. I am not a 100% sure how the NI savings work, as I have not received them myself, but my guess is it's up to the employer whether they want to include the apprentice levy or not. I wonder how common of a practice this is. It might be easier to make the 13.8 (or whatever the employer's NI rate is for a given year) a variable that can be changed by the user.

**Comment by jrothlis** [comment]
I agree that it would be simpler to make it a variable. If what the linked article says is right, it's not optional to have to pay the apprentice levy (if you're a company above a certain size) -- in the case of my company, they're being nice and letting me have the savings they would otherwise see by me sacrificing some of my salary or bonus.
